### DISCUSSION TITLE
Maintenance Mode doesnt Work

### DISCUSSION THREAD

**Discussion by levnorth** [givanz/Vvveb#151] [General]
I can switch between Maintenance, Live, and coming soon but the page doesnt change and even though it says maintenance in the toolbar, when I view the site its still in live mode.  I have cleared the cache and tried it in a private browser as well.

**Comment by givanz** [comment]
Do you have multiple sites or a single site configured? 

Can you please check if the site has the state set to maintenance in [config/sites.php](https://github.com/givanz/Vvveb/blob/master/config/sites.php#L28)

```php
<?php
 return array (
  '* * *' => 
  array (
    'name' => 'Default',
    'host' => '*.*.*',
    'theme' => 'landing',
    'state' => 'maintenance',// <<===
    'template' => '',
    'id' => 1,
    'site_id' => 1,
  ),
);
```
For admin users it will show the live version even if in maintenance mode.

For private browser (incognito) it should always show the maintenance page, if it doesn't work please check if both admin and frontend are on the same subdomain.

If  [V_SHARED_SESSION](https://github.com/givanz/Vvveb/blob/master/env.php#L50)  is not set to true then www.example.com and [example.com](example.com) might have different sessions.

You can try changing `V_SHARED_SESSION` in [env.php](https://github.com/givanz/Vvveb/blob/master/env.php#L50)   to true and see if solves the issue.

```php
defined('V_SHARED_SESSION') || define('V_SHARED_SESSION', true);
```
